Scripts/Helpers/ProxyHelper.cs

```diff
@@ -53,6 +53,17 @@ public Transform previewOrigin
 		private Transform m_PreviewOrigin;
 
 		/// <summary>
+		/// The transform that the display/preview objects will be parented under
+		/// </summary>
+		public Transform fieldGrabOrigin
+		{
+			get { return m_FieldGrabOrigin; }
+		}
+
```

Comment:
Ah. I was curious about this. Everything got an extra line in the refactor, presumably because R# does that when it auto-formats. Should I delete these extra lines where I find them?